test/s3-benchmarks/.scripts/benchmark

```diff
@@ -64,7 +64,7 @@ if [ ! -d result ]; then
 fi
 
 sizes_str="1B 8MB+1 8MB-1 128MB 4GB 30GB"
-versions_str="v1 v2 CRT"
+versions_str="v1 v2 CRT java"
```

Comment:
How does V2 differ from CRT and Java?

Reply:
`v2` is a benchmark for Transfer Manager using the CRTS3AsyncClient. `java` is using the new java-based s3 Multipart client

Reply:
What is CRT?

Reply:
`crt` : Indicate to run the benchmark for the CRT's `S3Client`
`java` : Indicate to run the benchmark for the java based S3 Async Client (`MultipartS3AsyncClient` class)
`v2`: SDK v2 transfer manager (using `S3CrtAsyncClient` to delegate requests)
`v1`: SDK v1 transfer manager (using `AmazonS3Client` to delegate requests)

Reply:
Ahh, okay, thanks! Maybe we should record that somewhere if it isn't already. Sorry if it is.

Reply:
I updated the README to include it. Now that the `--version` cli argument has more options, it is not a bad idea to specify it more clearly.